order/supplyunit.go

```diff
@@ -21,6 +21,12 @@ func NewSupplyFromSats(sats btcutil.Amount) SupplyUnit {
 	return SupplyUnit(uint64(sats) / uint64(BaseSupplyUnit))
 }
 
+// RoundToNextSupplyUnit computes and rounds to the next whole number of supply
```

Comment:
Can we remove the TODO on L19 or is there still something more to do, @Roasbeef?

Reply:
Nope this should be good, was just a hunch to revisit to ensure things are consistent. I think we need to do a sweep w.r.t where we use either of these methods, we may want to unify and ensure we always round up.